src/components/analyses/listing/TableView.js

```diff
@@ -0,0 +1,366 @@
+}
+
+function Status(props) {
+    const { analysis, username, baseId, analysisUser, intl } = props;
+    const allowTimeExtn =
+        analysis.interactive_urls &&
+        analysis.interactive_urls.length > 0 &&
+        analysis.status === analysisStatus.RUNNING &&
+        username === analysisUser;
+    return (
+        <React.Fragment>
+            <Link component="button" id={build(baseId, ids.STATUS)}>
+                {analysis.status}{" "}
+            </Link>
+            {allowTimeExtn && (
+                <Tooltip title={getMessage("extendTime")}>
+                    <IconButton
+                        id={build(baseId, ids.BUTTON_EXTEND_TIME_LIMIT)}
+                        size="small"
+                        aria-label={formatMessage(intl, "extendTime")}
+                    >
+                        <HourGlass />
+                    </IconButton>
+                </Tooltip>
+            )}
+        </React.Fragment>
+    );
+}
+
+const columnData = [
+    {
+        id: ids.NAME,
+        name: "Name",
+        numeric: false,
+        enableSorting: true,
+        key: "name",
+    },
+    {
+        id: ids.OWNER,
+        name: "Owner",
+        numeric: false,
+        enableSorting: false,
+        key: "owner",
+    },
+    {
+        id: ids.APP,
+        name: "App",
+        numeric: false,
+        enableSorting: false,
+        key: "app",
+    },
+    {
+        id: ids.START_DATE,
+        name: "Start Date",
+        numeric: false,
+        enableSorting: true,
+        key: "startdate",
+    },
+    {
+        id: ids.END_DATE,
+        name: "End Date",
+        numeric: false,
+        enableSorting: true,
+        key: "enddate",
+    },
+    {
+        id: ids.STATUS,
+        name: "Status",
+        numeric: false,
+        enableSorting: true,
+        key: "status",
+    },
+];
+
+function TableView(props) {
+    const {
+        loading,
+        error,
+        username,
+        listing,
+        parentId,
+        baseId,
+        order,
+        orderBy,
+        selected,
+        handleRequestSort,
+        handleSelectAllClick,
+        handleClick,
+        handleInteractiveUrlClick,
+        handleGoToOutputFolder,
+        handleBatchIconClick,
+        intl,
+    } = props;
+
+    const analyses = listing?.analyses;
+    const tableId = build(baseId, ids.LISTING_TABLE);
+
+    return (
+        <TableContainer
+            component={Paper}
+            style={{
+                height: "60vh",
+            }}
+        >
+            <Table
+                id={tableId}
+                stickyHeader={true}
+                size="small"
+                aria-label={formatMessage(intl, "ariaTableListing")}
+            >
+                <EnhancedTableHead
+                    baseId={baseId}
+                    selectable={true}
+                    numSelected={selected.length}
+                    rowsInPage={listing?.analyses ? listing.analyses.length : 0}
```

Comment:
This could be condensed (it's a little tricky to read as-is):
```suggestion
                    rowsInPage={listing?.analyses?.length || 0}
```